server/routes/changeEstablishment.ts

```diff
@@ -0,0 +1,20 @@
+import type { RequestHandler, Router } from 'express'
+
+import asyncMiddleware from '../middleware/asyncMiddleware'
+
+export default function routes(router: Router): Router {
+  const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+
+  const prisonList = [
```

Comment:
I am being picky but can this not be `enabledPrisons` too? Just makes the naming consistent since it is a list of enabled prisons.

Reply:
You must be telepathic Paul - suggested the same name when chatting to Ben just now 😂 !